packages/codegen/src/EntityDbMetadata.ts

```diff
@@ -700,3 +718,9 @@ function superstructType(s: string): Import {
   const [symbol, path] = s.split("@");
   return Import.from(`${pascalCase(symbol)}@${path}`);
 }
+
+function userFieldTypeType(s: string): Import {
+  // Assume it's `foo@...`, turn it into `Foo@...`
+  const [symbol, path] = s.split("@");
+  return Import.from(`${pascalCase(symbol)}@${path}`);
```

Comment:
I think we could probably skip this `pascalCase` / "assume it's foo -> Foo" step--that was primarily for superstruct, where there is both a runtime const `ipAddress` as well as type system `IpAddress`, and we wanted to have one config field that pointed to both.

For the `fieldType`, I think it'll always just be the type, like as you've got in the test `type: "IpAddress@..."`, so we can expect the user to have already camel-cased it/be pointing to a type.